force_wfmanager/server/zmq_server.py

```diff
@@ -229,13 +229,13 @@ def _close_network_sockets_noexc(self):
         self.ports = None
         try:
             self._pub_socket.close()
-        except:
+        except Exception:
```

Comment:
Some would argue this is ugly. But I'm not changing behavior here (and this fits the docstring "_This method throws away all exceptions that the operation might encounter_")

Reply:
Agreed: _Explicit is better than implicit._

Reply:
(to be clear: the change was dictated by Flake8, I just felt the need to justify myself for having used an _equally ugly_ `except`, where every exception -- including, I believe, deliberate keyboard interrupts -- is caught without scrutiny)